src/components/report-suspicious-activity/report-suspicious-activity-routes.ts

```diff
@@ -0,0 +1,23 @@
+import * as express from "express";
+import { reportSuspiciousActivityGet, reportSuspiciousActivityPost } from "./report-suspicious-activity-controller";
+import { PATH_DATA } from "../../app.constants";
+import { asyncHandler } from "../../utils/async";
+import { requiresAuthMiddleware } from "../../middleware/requires-auth-middleware";
+import { refreshTokenMiddleware } from "../../middleware/refresh-token-middleware";
+
+const router = express.Router();
+
+router.get(
+  PATH_DATA.REPORT_SUSPICIOUS_ACTIVITY.url,
```

Comment:
was just thinking about this, instead of using a query parameter, could we do something like
```suggestion
  `${PATH_DATA.REPORT_SUSPICIOUS_ACTIVITY.url}/:event_id`,
```
It wouldn't make a huge difference, but the URLs would be a bit nicer looking, and maybe a bit more semantic?

Reply:
There are more changes than this required to do this so will postpone until next ticket where I will look at the urls again.